docs/get-started/tooling/social-login/arcana.mdx

```diff
@@ -3,29 +3,19 @@ title: Arcana
 image: /img/socialCards/arcana.jpg
 ---
 
-[Arcana Network](https://arcana.network) SDKs enable web3 developers to use [social login](https://docs.arcana.network/concepts/social-login) 
-to onboard app users. 
+## Social Login
 
-Authenticated users can instantly access the embedded, non-custodial [Arcana wallet](https://docs.arcana.network/concepts/anwallet/) 
-from within the app context and sign blockchain transactions.
+[Arcana Auth SDK](https://arcana.network/auth) enables web3 developers to onboard app users quickly via [social login](https://docs.arcana.network/concepts/social-login). It enables the in-app, non-custodial [Arcana wallet](https://docs.arcana.network/concepts/anwallet/) for authenticated users to sign blockchain transactions. 
 
-Developers can leverage the built-in gasless feature to sponsor gas fees for allowlisted blockchain 
-transactions. The gasless transactions can be enabled for Arcana Wallet and third-party, 
-browser-based wallets.
+To get started, see the [Auth Quickstart Guide](https://docs.arcana.network/quick-start/) and follow the steps to integrate Web3 app. Use the appropriate SDK flavor as per your [app type](https://docs.arcana.network/sdk-installation). 
```

Comment:
```suggestion
To get started, see the [Auth Quickstart Guide](https://docs.arcana.network/quick-start/) and follow the steps to integrate a web3 app. Use the appropriate SDK flavor as per your [app type](https://docs.arcana.network/sdk-installation). 
```

Reply:
@jlwllmr can we use Web3.0 instead? In our docs we have been using Web3 but in consensys style guide I guess web3/Web3.0 is acceptable? Lmk. Thx

Reply:
we use `web3`

Reply:
ok, changing to web3. thx